### PR TITLE
Feature/LHUB023

### PR DESCRIPTION
**Pull request by guilhermedevel**
Create get page endpoint

### CODE REVIEW

File: src/main/java/com/godev/linkhubservice/domain/vo/CreatePageRequest.java [hunk 1]
```diff
@@ -36,19 +36,19 @@ public class CreatePageRequest {
     @Length(min = 4, max = 100, message = TITLE_LENGTH_ERROR)
     private String title;
 
-    @Schema(name = "description", defaultValue = "Link hub page description.", description = "A description of the page")
+    @Schema(name = "description", defaultValue = "Link Hub page description.", description = "A description of the page")
     @Length(min = 20, max = 200, message = DESCRIPTION_LENGTH_ERROR)
     private String description;
 
-    @Schema(name = "photo", defaultValue = "Link hub page photo.", description = "A photo of the page")
+    @Schema(name = "photo", defaultValue = "default.png", description = "A photo of the page")
```
Comment (jecesario) [inline src/main/java/com/godev/linkhubservice/domain/vo/CreatePageRequest.java]: Evaluate to use imgur url as default value

File: src/main/java/com/godev/linkhubservice/domain/vo/UpdatePageRequest.java [hunk 2]
```diff
@@ -33,19 +33,19 @@ public class UpdatePageRequest {
     @Length(min = 4, max = 100, message = TITLE_LENGTH_ERROR)
     private String title;
 
-    @Schema(name = "description", defaultValue = "Link hub page description.", description = "A description of the page")
+    @Schema(name = "description", defaultValue = "Link Hub page description.", description = "A description of the page")
     @Length(min = 20, max = 200, message = DESCRIPTION_LENGTH_ERROR)
     private String description;
 
-    @Schema(name = "photo", defaultValue = "Link hub page photo.", description = "A photo of the page")
+    @Schema(name = "photo", defaultValue = "default.png", description = "A photo of the page")
```
Comment (jecesario) [inline src/main/java/com/godev/linkhubservice/domain/vo/UpdatePageRequest.java]: Evaluate to use imgur url as default value

File: src/test/java/com/godev/linkhubservice/rest/controllers/impl/PageControllerImplTest.java [hunk 3]
```diff
@@ -30,17 +32,11 @@
...
-import static com.godev.linkhubservice.domain.constants.ValidationConstants.SLUG_REQUIRED_ERROR;
-import static com.godev.linkhubservice.domain.constants.ValidationConstants.TITLE_LENGTH_ERROR;
-import static com.godev.linkhubservice.domain.constants.ValidationConstants.URL_OR_HEX_FORMAT_ERROR;
-import static com.godev.linkhubservice.domain.exceptions.IssueEnum.ARGUMENT_NOT_VALID;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static com.godev.linkhubservice.domain.constants.IssueDetails.ID_NOT_FOUND_ERROR;
+import static com.godev.linkhubservice.domain.constants.IssueDetails.USER_NOT_ALLOWED;
+import static com.godev.linkhubservice.domain.constants.ValidationConstants.*;
```
Comment (jecesario) [inline src/test/java/com/godev/linkhubservice/rest/controllers/impl/PageControllerImplTest.java]: Wildcard

File: src/main/java/com/godev/linkhubservice/domain/vo/CreatePageRequest.java (hunk 1 above)
Comment (guilhermedevel) [reply src/main/java/com/godev/linkhubservice/domain/vo/CreatePageRequest.java]: resolved

File: src/main/java/com/godev/linkhubservice/domain/vo/UpdatePageRequest.java (hunk 2 above)
Comment (guilhermedevel) [reply src/main/java/com/godev/linkhubservice/domain/vo/UpdatePageRequest.java]: resolved

File: src/test/java/com/godev/linkhubservice/rest/controllers/impl/PageControllerImplTest.java (hunk 3 above)
Comment (guilhermedevel) [reply src/test/java/com/godev/linkhubservice/rest/controllers/impl/PageControllerImplTest.java]: resolved

**Review by jecesario — APPROVED** [review]

**Pull request merged**